format/spec.md

```diff
@@ -648,6 +648,9 @@ Notes:
 5. The `content_offset` and `content_size_in_bytes` fields are used to reference a specific blob for direct access to a deletion vector. For deletion vectors, these values are required and must exactly match the `offset` and `length` stored in the Puffin footer for the deletion vector blob.
 6. The following field ids are reserved on `data_file`: 141.
 
+For `variant` type, the `lower_bounds` and `upper_bounds` store the minimum and maximum values for all shredded subcolumns within a file. These bounds are represented as a Variant object, where each subcolumn path serves as a key and the corresponding bound value as the value. The object is then serialized into binary format (see [Variant encoding](https://github.com/apache/parquet-format/blob/master/VariantEncoding.md)).
```

Comment:
Since these are bounds and not true minimum/maximum (correct me if I'm wrong if variant shredding is somehow different), should we change the wording to simply say "store the lower and upper bounds for all shredded sumbcolumns within a file"?

Reply:
It means the same as the other bounds and it may not be the actual min/max. Let me update that.